config/ember-try.js

```diff
@@ -1,89 +1,83 @@
-'use strict'
+'use strict';
 
-const getChannelURL = require('ember-source-channel-url')
-const isCI = Boolean(process.env.CI)
+const isCI = !!process.env.CI;
```

Comment:
Should we apply the same change natively applied by Ember.js in the file `/tests/dummy/config/targets.js`?

```suggestion
const isCI = Boolean(process.env.CI);
```